app/models/tokite/repository.rb

```diff
@@ -32,10 +54,28 @@ def self.hooks_url
       Tokite::Engine.routes.url_helpers.hooks_url
     end
 
+    def self.owner(repo)
+      repo.split('/')[0]
+    end
+
     def unhook!(octokit_client)
-      hook = self.class.find_hook(octokit_client, name)
+      if is_org
+        unhook_org!(octokit_client)
+      else
+        unhook_repo!(octokit_client)
+      end
```

Comment:
I think these branches (organization or repository) are less readable.
How about splitting it into Tokite::Repository and Tokite::Organization?